src/LinuxTracing/GpuTracepointVisitor.cpp

```diff
@@ -128,33 +127,33 @@ void GpuTracepointEventProcessor::CreateGpuExecutionEventIfComplete(const Key& k
 // map and then try to create a complete GPU execution event. This event is only
 // created when all three types of GPU events have been received.
 
-void GpuTracepointEventProcessor::PushEvent(const AmdgpuCsIoctlPerfEvent& sample) {
```

Comment:
Can the parameter be const*? Same below.

Reply:
This is the signature of the method as defined in `PerfEventVisitor`.

Reply:
Right. Well should be fine.